tests/test_otel.py

```diff
@@ -258,6 +259,7 @@ def test_batches(client, trace_service, batch_count):
 
     for batch in trace_service.batches:
         assert get_attr(batch[0].resource, "service.name") == "test_service"
+        assert get_attr(batch[0].resource, "my.name") == "my service"
```

Comment:
This might deserve a separate test, but can keep testing for default service name ("unknown_service:nginx") here.
Also, let's have two attributes added, to ensure the directive can be used several times.

Reply:
Added a separate test.